src/app/bav/controllers/confirmDetails.js

```diff
@@ -43,6 +43,9 @@ class ConfirmDetailsController extends BaseController {
     const res = await axios.post(`${API.PATHS.SAVE_BAVDATA}`, bavData, {
       headers,
     });
+    if (res.data.retryCount) {
+      req.sessionModel.set("retryCount", res.data.retryCount);
+    }
```

Comment:
Could we cover this with a unit test please?

Reply:
Added ✅